tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/IastVerifyScrubberExtensions.cs

```diff
@@ -16,19 +16,22 @@ public static class IastVerifyScrubberExtensions
         private static readonly Regex NetworkClientIp = new(@"["" ""]*network.client.ip: .*,(\r|\n){1,2}");
         private static readonly Regex HashRegex = new(@"(\S)*""hash"": (-){0,1}([0-9]){1,12},(\r|\n){1,2}      ");
         private static readonly Regex RequestTaintedRegex = new(@"_dd.iast.telemetry.request.tainted:(\s)*([1-9])(\d*).?(\d*),");
+        private static readonly Regex TelemetryExecutedSinks = new(@"_dd\.iast\.telemetry\.executed\.sink\.weak_.+: .{3},");
```

Comment:
Do we need _any_ of the sink metrics 🤔 Should we just scrub them all?

Reply:
Yes, we check that the sink teste in the particular test (not weak_whatsoever) is present...